src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -245,6 +247,23 @@ private static boolean isSameRequester(ConsentArtefact consentDetail, String req
                                                 .thenReturn(consentApprovalResponse(consents)))));
     }
 
+    private Mono<Void> validateDate(List<GrantedConsent> grantedConsents){
+        Optional<GrantedConsent> falseValidation = grantedConsents.stream()
+                .filter(grantedConsent -> dateCheckerForNullAndFuture(grantedConsent) == false).findAny();
```

Comment:
- Use `stream().allMatch()` instead of filter.
- Rename the variable `falseValidation`

Reply:
Changing that. Thanks